notebooks/02-Introduction.ipynb

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
<p>Block index should be mentioned.</p><p> </p><p>Also is there any way to show a "diagram" of the low-level file layout, kind of how we now display the files in the asdf documentation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/asdf-format/scipy2022tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>